hbase-mapreduce/src/test/java/org/apache/hadoop/hbase/mapred/TestTableInputFormat.java

```diff
@@ -147,7 +147,8 @@ public static Table createTable(byte[] tableName, byte[][] families) throws IOEx
    * @param key
    * @param expectedKey
    * @param expectedValue
-   * @return true if succeed
+   * @return true if result and key have expected values, false otherwise. Where result is a single
```

Comment:
Better 'true if the result contains the expected key and value, false otherwise'. The second sentence is not necessary and is something, which I would expect in the description of the `r` parameter.